pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+requires = [
+  "setuptools>=42",
+  "scikit-build",
+  "pybind11",
```

Comment:
What is the rationale for adding pybind11 here? Shouldn't we expect pybind11 to be installed with the other of C++ dependencies?